.gitignore

```diff
@@ -4,6 +4,6 @@
 .vscode
 
 # infra binary
-dist
-docs/api/openapi3.json
```

Comment:
This intentionally removed from the gitignore?

Reply:
Yes. The ignore on `dist` was recursive which erroneously ignored the actions' `dist` directories. Removing the openapi spec is intentional as well because the file has been moved.